File: DailyFlash Solutions/DailyFlash Solutions/01-DailyFlash_Solutions/14_Jan_Solutions_One/C++/prog5.cpp
#include<iostream>

int main() {
	int n1, n2, n3;
	std::cout<<"Input : ";
	std::cin>>n1>>n2>>n3;
	n1>n2 && n1>n3 ? std::cout<<"Output : "<<n1:n2>n3 ? std::cout<<"Output : "<<n2:std::cout<<"Output : "<<n3;
	return 0;
}

File: DailyFlash Solutions/DailyFlash Solutions/01-DailyFlash_Solutions/14_Jan_Solutions_One/Java/prog5.java
import java.io.*;
class Demo{
	public static void main(String[] args) throws IOException {
		BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
		System.out.println("Input : ");
		int n1 = Integer.parseInt(br.readLine());
		int n2 = Integer.parseInt(br.readLine());
		int n3 = Integer.parseInt(br.readLine());
		System.out.println(n1>n2 && n1>n3?"Output : " + n1:n2>n3?"Output : " + n2:"Output : " + n3);
	}
}
